edi_af_appointment/controllers/main.py

```diff
@@ -177,166 +246,218 @@ def reserve_bookable_occasion_json(self, bookable_occasion_id=False, **kwargs):
-                "office_address": app.office_id.partner_id.contact_address or '',
-                "office_email": app.office_id.partner_id.email or '',
-                "location_code": app.operation_id.location_code or '',
-                "office_code": app.office_id.office_code or '',
-                "office_name": app.office_id.display_name or '',
+                "office_address": app.office_id.partner_id.contact_address or "",
+                "office_email": app.office_id.partner_id.email or "",
+                "location_code": app.operation_id.location_code or "",
+                "office_code": app.office_id.office_code or "",
+                "office_name": app.office_id.display_name or "",
                 "status": app.state,
             }
             return res
         else:
             return Response("ID not found", status=404)
 
-    @http.route('/v1/appointments/bookable-occasions/reservation/<bookable_occasion_id>', type='http', csrf=False, auth="public", methods=['DELETE'])
+    @http.route(
+        "/v1/appointments/bookable-occasions/reservation/<bookable_occasion_id>",
+        type="http",
+        csrf=False,
+        auth="public",
+        methods=["DELETE"],
+    )
     def unreserve_bookable_occasion(self, bookable_occasion_id=False, **kwargs):
         # TODO: fix these error messages
         occasions = self.decode_bookable_occasion_id(bookable_occasion_id)
         if not occasions:
             return Response("ID not found", status=404)
         try:
-            if request.env['calendar.appointment'].sudo().delete_reservation(occasions):
+            if request.env["calendar.appointment"].sudo().delete_reservation(occasions):
                 return Response("OK, reservation deleted", status=200)
             else:
                 return Response("ID not found", status=404)
         except:
             return Response("ID not found", status=404)
 
-    @http.route('/v1/appointments/appointments', type='http', auth="public", methods=['GET'])
-    def get_appointment(self, user_id=False, customer_nr=False, pnr=False, appointment_types=False, status_list=False, start=False, stop=False, **kwargs):
+    @http.route(
+        "/v1/appointments/appointments", type="http", auth="public", methods=["GET"]
+    )
+    def get_appointment(
+        self,
+        user_id=False,
+        customer_nr=False,
+        pnr=False,
+        appointment_types=False,
+        status_list=False,
+        start=False,
+        stop=False,
+        **kwargs
+    ):
         search_domain = []
         partner = False
 
-        if not (user_id or customer_nr or pnr or appointment_types or status_list or start or stop):
+        if not (
+            user_id
+            or customer_nr
+            or pnr
+            or appointment_types
+            or status_list
+            or start
+            or stop
+        ):
             return Response("No arguments given.", status=400)
 
         if pnr:
```

Comment:
If both pnr and customer_nr is supplied would be an error or do we want to silently swallow it and go for pnr as we do here?